source/sync/data-model/update-schema.txt

```diff
@@ -26,443 +21,518 @@ Update Your Data Model
 
 .. toctree::
    :titlesonly:
+   
+   Update with Development Mode </sync/data-model/update-data-model/update-schema-development-apps>
+   Update with Schema Versioning </sync/data-model/update-data-model/update-schema-production-apps>
+   Make Breaking Changes </sync/data-model/update-data-model/migrate-schema-partner-collection>
+
+.. TODO: After date is confirmed, update [DATE TBD] throughout project
+
+When developing an application using Atlas Device Sync, you may need to make 
+changes to your Device Sync data model. For example,
+you might need a new object type or decide that an optional property should be 
+required.
+
+In general, we refer to data model updates in the following terms: 
+
+*What type of change are you making?* 
+
+We broadly categorize changes to a Device Sync data model
+as being either breaking or non-breaking. Breaking changes, such as changing
+the data type of an existing property, are considered destructive and require
+additional handling to apply. These are discussed in more detail below, or you
+can check out the :ref:`breaking-change-quick-reference` section on this page. 
+
+*Where are you making the change?* 
+
+Because the Device Sync data model consists of a client-side SDK object model
+and a server-side App Services schema, we distinguish between making changes
+to the client side (updating the SDK object models defined in your client
+code) or the server side (updating the collection schema through the App
+Services UI or Admin API). Regardles of where you make your
+changes, though, you must apply those same changes to the other side to ensure
+both sides remain consistent. The implications of where you make your changes are discussed in more detail below.
+
+*Is Development Mode or schema versioning enabled?* 
+
+When enabled for your app, the following Device Sync features can simplify how you update your data model:  
+
+- **Development Mode:** Development Mode is a Device Sync configuration setting.
+  When enabled, you can make breaking changes from the client side, and App
+  Services automatically handles updating the server-side schema. Apps must be
+  created after September 13, 2023 and meet a minimum Device SDK version. For
```

Comment:
IMO this makes it seem like users can't use dev mode if the app is older than this. This date is just the cutoff for the "client-side breaking changes" feature. Older clients can still use dev mode without that feature

Reply:
ah, yeah. It was originally only referring to the breaking changes in Dev Mode, but it got blurred a few revisions ago. I'll remove it here since it's potentially confusing. thanks for catching